pkg/store/mongodb/storeBuiler.go

```diff
@@ -0,0 +1,83 @@
+//Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+//Licensed under the Universal Permissive License (UPL) Version 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+package mongodb
+
+import (
+	"context"
+
+	"github.com/prometheus/common/log"
```

Comment:
We are using logrus, not prometheus log.

Reply:
will fix it.